### ISSUE TITLE
cannot compile?

### ISSUE THREAD

**Issue by meanjellybean1** [matterpreter/DefenderCheck#25]
have this error: System.IndexOutOfRangeException: 'Index was outside the bounds of the array.'

**Comment by MehdiBHA** [comment]
same 
anyway to fix this? @matterpreter

**Comment by matterpreter** [comment]
This sounds like a runtime error, not an error during compilation. Could you please provide the full stack trace?

**Comment by nothing4free** [comment]
Hi! I'm having the same error on my side. Compiled using Visual Studio 2022, the full stack trace I'm getting is the following:
```
.\DefenderCheck.exe
System.IndexOutOfRangeException: 'Index was outside the bounds of the array.' at DefenderCheck.Program.Main(String[] args) at C:\Users\n4f\Desktop\DefenderCheck-master\DefenderCheck\DefenderCheck\Program.cs:line 21
```

**Comment by matterpreter** [comment]
Fixed in [27616de](https://github.com/matterpreter/DefenderCheck/commit/27616dea8d27a9d926f5b2178b114109f482c60b)
